ballista-examples/examples/test_sql.rs

```diff
@@ -0,0 +1,46 @@
+// Licensed to the Apache Software Foundation (ASF) under one
+// or more contributor license agreements.  See the NOTICE file
+// distributed with this work for additional information
+// regarding copyright ownership.  The ASF licenses this file
+// to you under the Apache License, Version 2.0 (the
+// "License"); you may not use this file except in compliance
+// with the License.  You may obtain a copy of the License at
+//
+//   http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing,
+// software distributed under the License is distributed on an
+// "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+// KIND, either express or implied.  See the License for the
+// specific language governing permissions and limitations
+// under the License.
+
+use ballista::prelude::{BallistaConfig, BallistaContext, Result};
+use datafusion::prelude::CsvReadOptions;
+
+/// This example show the udf plugin is work
+///
+#[tokio::main]
+async fn main() -> Result<()> {
+    let config = BallistaConfig::builder()
+        .set("ballista.shuffle.partitions", "1")
+        .build()?;
+
+    let ctx = BallistaContext::standalone(&config, 10).await.unwrap();
+
+    let testdata = datafusion::test_util::arrow_test_data();
+
+    // register csv file with the execution context
+    ctx.register_csv(
+        "aggregate_test_100",
+        &format!("{}/csv/aggregate_test_100.csv", testdata),
+        CsvReadOptions::new(),
+    )
+    .await?;
+
+    // test udf
+    let df = ctx.sql("select count(1) from aggregate_test_100").await?;
```

Comment:
💯  for test

Reply:
@alamb 
Do we have the UT or IT for the ballista side?
I think this `test_sql.rs` file just adds an example for the ballista.

Reply:
Maybe we can add this test for the UT or it IT framework.

Reply:
@liukun4515  I think we need a ballista-integration-test cargo.

Reply:
@liukun4515  How about add a ballista-integration-test cargo?

Reply:
I think https://github.com/apache/arrow-datafusion/issues/321 tracks some of this -- I will make a note

Reply:
> about add a ballista-integration-

agree with the suggestion.
we can discuss this in this issue https://github.com/apache/arrow-datafusion/issues/321.

@gaojun2048  you can provide some suggestions or proposals.
You can point out some lack of ballista about test and record them.
We can make a plan to address them in Q2.